app/components/ChoiceDropdown/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Label } from 'reactstrap'
+import Select from 'react-select';
+
+import styles from './styles.css'
+
+class ChoiceDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      answer: undefined
+    }
+  }
+
+  render() {
+
+    let options = this.props.answers.map((answer, index) => {
+      return { value: index, label: answer }
+    })
+
+    let logChange = (val) => {
+      this.setState({
+        answer: val
```

Comment:
you shouldn't set state in a component unless you absolutely have to, redux should pass the component value via props